### PR TITLE
http: Handle / endpoint gracefully

### PR DESCRIPTION
**Pull request by mjtrangoni**
Hi @bobrik,

this trivial PR, 
- handle the `/` endpoint gracefully (it returned a 404 before).
- do some aesthetic changes on the help output.
- make the `/metrics` endpoint configurable.

Most of the changes are in-line with the [node_exporter](https://github.com/prometheus/node_exporter) style, and I hope this is ok for you.

See,

```html
 <html>
 <head><title> eBPF Exporter</title></head>
 <body>
  <h1>eBPF Exporter</h1>
  <p><a href="/metrics">Metrics</a></p>
  </body>
 </html>
```

and,
```bash
$ ./ebpf_exporter --help
usage: ebpf_exporter [<flags>]

Flags:
  -h, --help                     Show context-sensitive help (also try --help-long and --help-man).
      --config.file=config.yaml  Config file path.
      --debug                    Enable debug.
      --web.listen-address=":9435"  
                                 The address to listen on for HTTP requests.
      --web.telemetry-path="/metrics"  
                                 Path under which to expose metrics.
      --version                  Show application version.
```

### CODE REVIEW

File: cmd/ebpf_exporter/main.go [hunk 1]
```diff
@@ -41,7 +52,19 @@ func main() {
 		log.Fatalf("Error registering exporter: %s", err)
 	}
 
-	http.Handle("/metrics", promhttp.Handler())
+	http.Handle(*metricsPath, promhttp.Handler())
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		num, err = w.Write([]byte(`<html>
+			<head><title> eBPF Exporter</title></head>
```
Comment (bobrik) [inline cmd/ebpf_exporter/main.go:58]:
```suggestion
			<head><title>eBPF Exporter</title></head>
```

File: cmd/ebpf_exporter/main.go [hunk 2]
```diff
@@ -41,7 +52,19 @@ func main() {
...
+			<body>
+			<h1>eBPF Exporter</h1>
+			<p><a href="` + *metricsPath + `">Metrics</a></p>
+			</body>
+			</html>`))
+		if err != nil {
+			log.Fatal(num, err)
```
Comment (bobrik) [inline cmd/ebpf_exporter/main.go:65]:
```suggestion
			log.Fatalf("Error sending response body: %s", err)
```

File: cmd/ebpf_exporter/main.go [hunk 3]
```diff
@@ -41,7 +52,19 @@ func main() {
 		log.Fatalf("Error registering exporter: %s", err)
 	}
 
-	http.Handle("/metrics", promhttp.Handler())
+	http.Handle(*metricsPath, promhttp.Handler())
+	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+		num, err = w.Write([]byte(`<html>
```
Comment (bobrik) [inline cmd/ebpf_exporter/main.go:57]:
```suggestion
		_, err = w.Write([]byte(`<html>
```

File: cmd/ebpf_exporter/main.go [hunk 4]
```diff
@@ -14,9 +14,20 @@ import (
 )
 
 func main() {
-	listenAddress := kingpin.Flag("web.listen-address", "The address to listen on for HTTP requests").Default(":9435").String()
-	configFile := kingpin.Flag("config.file", "Config file path").Default("config.yaml").File()
-	debug := kingpin.Flag("debug", "Enable debug").Bool()
+	var (
```
Comment (bobrik) [inline cmd/ebpf_exporter/main.go:17]: I'd rather keep this block as it was.

**Review by bobrik — CHANGES_REQUESTED** [review]

**Comment by mjtrangoni** [comment]
@bobrik thanks for the review. PTAL.

File: cmd/ebpf_exporter/main.go [hunk 5]
```diff
@@ -14,9 +14,10 @@ import (
 )
 
 func main() {
-	listenAddress := kingpin.Flag("web.listen-address", "The address to listen on for HTTP requests").Default(":9435").String()
-	configFile := kingpin.Flag("config.file", "Config file path").Default("config.yaml").File()
-	debug := kingpin.Flag("debug", "Enable debug").Bool()
+	configFile := kingpin.Flag("config.file", "Config file path.").Default("config.yaml").File()
+	debug := kingpin.Flag("debug", "Enable debug.").Bool()
```
Comment (bobrik) [inline cmd/ebpf_exporter/main.go:18]: Let's not have dots in the description. LGTM otherwise.

Comment (mjtrangoni) [reply cmd/ebpf_exporter/main.go]: fixed

**Comment by bobrik** [comment]
Thanks!

**Pull request merged**